installer/ubuntu/bootstrap

```diff
@@ -46,6 +46,14 @@ else
     echo "Unable to patch debootstrap, moving on..." 1>&2
 fi
 
+# bionic onwards does not require installing apt-transport-https (apt provides
+# it). In theory, there is a transitional package, but debootstrap does not find
+# it. Somewhat relevant bug:
+#   https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=879755
+if release -ge bionic; then
```

Comment:
Seems to also affect buster and sid?  And probably kali-rolling?

Reply:
For some reason debootstrap does find the transitional package on buster and sid. Unsure about kali-rolling (our tests are using an http server...).

We could file a bug for bionic...

Reply:
Maybe the transitional package was added late in bionic and ended up in the wrong repo.  But it sounds like this package is history and we should remove it if `! release -le artful -le stretch -le kali` or something?

Reply:
Well, hopefully debootstrap will end up fixing this...

But, yeah, I can make the rule more general and try to clean that up in one of the future uprevs.